packages/rule-components/src/RuleTable/index.scss

```diff
@@ -1,7 +1,7 @@
-@import '~@redhat-cloud-services/frontend-components/components/battery.scss';
-@import '~@redhat-cloud-services/frontend-components/components/skeleton.css';
-@import '~@redhat-cloud-services/frontend-components/components/TableToolbar.scss';
-@import '~@redhat-cloud-services/frontend-components/components/primary-toolbar.scss';
+@import '~@redhat-cloud-services/frontend-components/components/Battery.css';
```

Comment:
Is there any reason why you moved to css instead of scss? It should be better to use sass files and css as fallback.

Reply:
I did this because when I did `npm start` it showed errors that, this files are not found.

Reply:
There were couple of big updates, recently. You should run `npm run clean; npm install` (in the roor folder) to clean and install the new versions

Reply:
Ok, thank you